edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.IoTHub.Test/EdgeAgentConnectionTest.cs

```diff
@@ -341,7 +341,7 @@ static async Task<Device> CreateEdgeDeviceWithCondition(string edgeDeviceId, Reg
         }
 
         [Integration]
-        [Fact(Skip = "Flaky")]
+        [Fact]
```

Comment:
I think we talked about that one. It is not flaky, but long running. I thought that we were going to change that to pull frequently, instead just pausing for 7 min?